test/functional/test-mediasession-helper.js

```diff
@@ -141,6 +141,7 @@ describes.sandboxed('MediaSessionAPI Helper Functions', {}, () => {
       'artist': '',
       'album': '',
       'artwork': [
+        /*eslint no-script-url: 0*/
```

Comment:
I think this disables for whole file so no need for second `eslint no-script-url` right?